src/backend/commitrange.h

```diff
@@ -23,22 +25,29 @@ namespace snmalloc
       }
     };
 
+    using B = typename ParentRange::B;
+    using KArg = typename ParentRange::KArg;
+
     static constexpr bool Aligned = ParentRange::Aligned;
 
     constexpr CommitRange() = default;
 
-    capptr::Chunk<void> alloc_range(size_t size)
+    CapPtr<void, B> alloc_range(KArg ka, size_t size)
```

Comment:
Making `KArg ka` the last parameter might be a useful trick, so we can add defaults and not change the API.